src/main/java/ch/wisv/events/model/Event.java

```diff
@@ -46,6 +46,10 @@ public long getId() {
         return id;
     }
 
+    public void setId(long id) {
+        this.id = id;
+    }
+
```

Comment:
I still think that this is a better way than copying every entry from a new Event to the existing one. It's much cleaner imo.